### PR TITLE
Feature/2.7encode

### PR DESCRIPTION
**Pull request by defond0**
Add future as requirement and use it to allow for decoding in 2.7 and 3 for kms and gpg.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -15,7 +15,8 @@
     'boto3',
     'gnupg>=2.0.2',
     'seria',
-    'python-gnupg'
+    'python-gnupg',
+    'future'
```
Comment (theherk) [inline setup.py]: Alpha order is best order.

File: figgypy/decrypt.py [hunk 2]
```diff
@@ -1,8 +1,9 @@
+from __future__ import unicode_literals
+from future.utils import bytes_to_native_str as n
 """Decrypt objects in Config."""
 import base64
 import logging
 import os
-
```
Comment (theherk) [inline figgypy/decrypt.py:5]: I liked that space. :)

File: figgypy/decrypt.py [hunk 3]
```diff
@@ -1,8 +1,9 @@
+from __future__ import unicode_literals
+from future.utils import bytes_to_native_str as n
 """Decrypt objects in Config."""
```
Comment (theherk) [inline figgypy/decrypt.py]: Docstring to the top.

File: setup.py (hunk 1 above)
Comment (defond0) [inline setup.py]: true story, good eye

File: figgypy/decrypt.py (hunk 2 above)
Comment (defond0) [inline figgypy/decrypt.py:5]: then you shall have it sir :)

**Pull request merged**